templates/element/variables_panel.php

```diff
@@ -24,6 +24,14 @@
     printf('<p class="warning">%s</p>', $error);
 endif;
 
+if (isset($varsMaxDepth)) {
+    $msg = sprintf(__d('debug_kit', '%s levels of nested arrays shown.'), $varsMaxDepth);
```

Comment:
```suggestion
    $msg = sprintf(__d('debug_kit', '%s levels of nested data shown.'), $varsMaxDepth);
```

More than arrays gets limited.